src/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -790,10 +793,15 @@ private function registerArgumentResolverConfiguration(XmlFileLoader $loader): v
         $loader->load('argument_resolver.xml');
     }
 
-    private function triggerDeprecations(): void
+    private function registerInflectorConfiguration(array $config): void
     {
-        if (!class_exists(UnicodeString::class)) {
-            trigger_deprecation('api-platform/core', '3.2', 'Using doctrine/inflector is deprecated since API Platform 3.2 and will be removed in API Platform 4. Use symfony/string instead. Run "composer require symfony/string".');
+        if ($config['keep_legacy_inflector']) {
+            Inflector::keepLegacyInflector(true);
```

Comment:
```suggestion
```

No need to fix this one its deprecated we'll remove it in 4.x and it won't really be used once we have the subtree split